docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-syntax.md

```diff
@@ -1,4 +1,4 @@
-# SHOW 语法
+# SHOW 语句
```

Comment:
May be you need to update `SHOW VARIABLES`, and the doc link of it. So does the EN doc